backend/src/dao/TeamEventsDao.ts

```diff
@@ -1,107 +1,64 @@
 import { v4 as uuidv4 } from 'uuid';
-import { DBTeamEvent } from '../types/DataTypes';
-import { memberCollection, teamEventsCollection, bucket } from '../firebase';
+import { teamEventsCollection, bucket } from '../firebase';
 import { NotFoundError } from '../utils/errors';
-import { getMemberFromDocumentReference } from '../utils/memberUtil';
 
 export default class TeamEventsDao {
-  static async getAllTeamEvents(): Promise<TeamEvent[]> {
+  static async getAllTeamEvents(): Promise<TeamEventInfo[]> {
     const eventRefs = await teamEventsCollection.get();
     return Promise.all(
       eventRefs.docs.map(async (eventRef) => {
-        const { name, date, numCredits, hasHours, requests, attendees, uuid, isCommunity } =
-          eventRef.data();
+        const { name, date, numCredits, hasHours, uuid, isCommunity } = eventRef.data();
```

Comment:
+1. This can be rewritten as

`return eventRefs.docs.map(async (eventRef) => eventRef.data() as TeamEventInfo)`. No need for the `Promise.all(...)` either.

Reply:
I tried this, and it was giving a Promise<TeamEventInfo>[] without the Promise.all, so i kept it, but let me know how this looks!

Reply:
Oh yep, oops. You're right. And I would write it as
```
return Promise.all(eventRefs.docs.map(async (eventRef) => eventRef.data()));
```